tests/api/test_router_wrapper.py

```diff
@@ -0,0 +1,47 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+from regtech_api_commons.api.router_wrapper import Router
+
+router = Router()
+
+
+@router.api_route("/items/{item_id}", methods=["GET"])
+def get_items(item_id: str):
+    return {"item_id": item_id}
+
+
+def get_not_decorated(item_id: str):
+    return {"item_id": item_id}
+
+
+router.add_api_route("/items-not-decorated/{item_id}", get_not_decorated)
+
+app = FastAPI()
+app.include_router(router)
+
+client = TestClient(app)
+
+
+def test_get_api_route():
+    response = client.get("/items/foo")
+    assert response.status_code == 200, response.text
+    assert response.json() == {"item_id": "foo"}
+
+
+def test_get_api_route_ends_with_forward_slash():
+    response = client.get("/items/foo/")
+    assert response.status_code == 200, response.text
+    assert response.json() == {"item_id": "foo"}
+
+
+def test_get_api_route_wrong_path():
+    response = client.get("/item/foo/")
+    assert response.status_code != 200, response.text
+    assert response.json() != {"item_id": "foo"}
+
+
+def test_get_api_route_not_decorated():
```

Comment:
maybe add another test for the not decorated one, so we can see the response being 302 (I think, the redirect one)

Reply:
sorry... should've read the code closer; were you just trying to test out the different ways to hook up a route? I thought you were testing out the different behaviors of router wrapper and fastapi's built-in router. In that case you can remove the redirect test since it's a manually provided redirect.

Reply:
I was just trying to test whether the wrapper hits the same endpoint if a slash is added at the end of the url. I think the not decorated test is not needed here, so removed it. But I can more tests per your suggestion.